header/script.js

```diff
@@ -0,0 +1,65 @@
+window.Header = {
```

Comment:
I suggest you change the name of bind for `bindHeader`, because another components is: `openPage()`, `openMenu()`, etc